src/network/message.rs

```diff
@@ -154,7 +154,9 @@ pub enum NetworkMessage {
     /// `alert`
     Alert(Vec<u8>),
     /// `reject`
-    Reject(message_network::Reject)
+    Reject(message_network::Reject),
+    /// `feefilter`
+    FeeFilter(i64),
```

Comment:
Shouldn't this be unsigned?

Reply:
Yeah a negative fee rate never makes sense but the bip says int64_t?

Reply:
Right, its a bit confusing still - Bitcoin Core rejects/ignores feefilter messages if they are > 21 million BTC or < 0, even though the datatype is technically unsigned. It doesn't really matter much, though.